app/javascript/components/components/Header.js

```diff
@@ -15,45 +15,36 @@ class Header extends Component {
       new_user_route,
       sign_in_route,
       sign_out_route,
-      create_auction_route,
+      auction_new_route,
       my_auctions_route,
       my_bids_route
     } = this.props
 
     return (
       <>
         <header>
-        
-            <div className="nav-bar">
-              <NavLink to="/">
-                <img src={bidLogo} alt="Bid up logo" className="logo"/>
-              </NavLink>
-            </div>
-          
-          
-        
-            <div className="nav-bar">
-              {logged_in &&
-                <NavLink to= "/create_auction_route" className="nav-link">Create Auction</NavLink>
-              }
-              {logged_in &&
-                <NavLink to= "/my_auctions_route" className="nav-link">My Auctions</NavLink>
-              }
-              {logged_in &&
-                <NavLink to= "/my_bids_route" className="nav-link">My Bids</NavLink>
-              }
-            </div>
-      
-
-           <div className="nav-bar">
-            
-                <Nav>
-                  <NavItem>
-                    <NavLink to="/about" className="nav-link">About Us</NavLink>
-                  </NavItem>
-                </Nav>
-              
-
+          <div className="nav-bar">
+            <NavLink to="/">
+              <img src={bidLogo} alt="Bid up logo" className="logo"/>
+            </NavLink>
+          </div>
+          <div className="nav-bar">
+            {logged_in &&
+              <NavLink to= "/auction_new_route" className="nav-link">Create Auction</NavLink>
+            }
+            {logged_in &&
+              <NavLink to= "/my_auctions_route" className="nav-link">My Auctions</NavLink>
+            }
+            {logged_in &&
+              <NavLink to= "/my_bids_route" className="nav-link">My Bids</NavLink>
+            }
+          </div>
+          <div className="nav-bar">
+              <Nav>
+                <NavItem>
+                  <NavLink to="/about" className="nav-link">About Us</NavLink>
+                </NavItem>
+              </Nav>
```

Comment:
@hawk198723 I think you may have format on save enabled in VSCode. You, Kevin, and Raul should sync your VSCode settings so the diff on PR's in not as big. It looks like there were a lot of changes here, but looking at it, I think most of it was just indentation.

If you all install this VSCode extension: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode, then enable "Format on Save" in the VSCode settings, it should format all your code the same way.